tensorrt_llm/inputs/media_io.py

```
@@ -313,24 +341,72 @@ def extract_audio_from_video(
+        _api_pref = None
+        try:
+            for _backend in _vr.getStreamBufferedBackends():
+                if not _vr.hasBackend(_backend):
+                    continue
+                if not _vr.isBackendBuiltIn(_backend):
+                    _, _abi, _api = _vr.getStreamBufferedBackendPluginVersion(_backend)
+                    if _abi < 1 or (_abi == 1 and _api < 2):
+                        continue
+                _api_pref = _backend
+                break
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)